SECP-service/src/main/java/com/visucius/secp/models/PermissionLevel.java

```diff
@@ -0,0 +1,63 @@
+package com.visucius.secp.models;
+
+import javax.persistence.*;
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+
+@Entity
+@Table(name = "permissionLevel")
+public class PermissionLevel {
+
+    @Id
+    @GeneratedValue
+    @Column(name = "id", unique = true, nullable = false)
+    private long id;
+
+
+    @Column(name = "level", nullable = false)
+    private int level;
```

Comment:
This should be a string and also the column should be unique. Please take a look at how we did it for the Role class.